client/extensions/woocommerce/woocommerce-services/lib/utils/print-document.js

```diff
@@ -55,12 +55,24 @@ const buildBlob = ( b64Content, mimeType ) => {
  * @returns {Promise} Promise that resolves if the printing dialog (or equivalent) was correctly
  * invoked, rejects otherwise.
  */
-export default ( { b64Content, mimeType }, fileName ) => {
+export default ( { b64Content, mimeType }, fileName, download = false ) => {
```

Comment:
Please add much more comments here and in the `pdf-support.js` file. This entire logic is so hacky and full of cross-browser traps, than the only sensible way to follow it is with more comments than code.